pkg/api/graph/graph.go

```diff
@@ -41,8 +41,12 @@ func (n UniqueName) UniqueName() string {
 	return string(n)
 }
 
+func (n UniqueName) String() string {
+	return string(n)
+}
+
 type uniqueNamer interface {
-	UniqueName() string
+	UniqueName() UniqueName
```

Comment:
On Tue, Apr 26, 2016 at 1:47 PM, Clayton Coleman <notifications@github.com>
wrote:

> In pkg/api/graph/graph.go
> <https://github.com/openshift/origin/pull/8351#discussion_r61131701>:
>
> >  type uniqueNamer interface {
> > -	UniqueName() string
> > +	UniqueName() UniqueName
>
> There is no point to this change?
>
Actually, this change was needed to properly line up implementations of
this interface and
https://github.com/openshift/origin/blob/master/pkg/api/graph/graph.go#L122-L129
,
My testing found cases where
https://github.com/openshift/origin/blob/master/pkg/api/graph/graph.go#L127
was being called instead of UniqueName()

> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8351/files/565f03e9e5bdf023cb7792c61816f27c70e84707#r61131701>
>